_layouts/home.html

```diff
@@ -2,7 +2,6 @@
 layout: default
 refactor: true
 ---
```

Comment:
 Please leave one blank line at the bottom of the font-matter.

I'm assuming this was deleted by the editor when it was auto-formatted; to avoid this, you can leave two blank lines at the bottom of the font before saving the file.

```suggestion
---

```